modules/market/src/main/java/com/opengamma/strata/market/curve/CurveInfoType.java

```diff
@@ -38,11 +38,11 @@
   /**
    * Obtains an instance from the specified name.
    * <p>
-   * Curve names may contain any character, but must not be empty.
+   * The name may contain any character, but must not be empty.
    *
-  * @param <T>  the type associated with the info
-   * @param name  the name of the curve
-   * @return a curve with the specified name
+   * @param <T>  the type associated with the info
+   * @param name  the name
+   * @return a type instance with the specified name
    */
   @FromString
   public static <T> CurveInfoType<T> of(String name) {
```

Comment:
I find it very surprising that this class has a type parameter that is never used anywhere. It contains no instances of that type and no `Class` object of the type. It's like a compile-time phantom which disappears and can never be used at runtime.